x/twap/logic_test.go

```diff
@@ -1558,6 +1592,33 @@ func (s *TestSuite) TestAfterCreatePool() {
 	}
 }
 
+func (s *TestSuite) TestTwapLog() {
```

Comment:
Note: these are moved to group tests at the top of the file and helpers at the bottom. These are unchanged

Reply:
Ok... , don't think this is an important code style thing though